test/benchmarks/003-alice/ppbench/Config.C

```diff
@@ -1314,28 +1314,28 @@ AliGenerator* GeneratorFactory(PprRun_t srun) {
         break;
       case kFlow_2_2000:
       {
-        comment = comment.Append(" Flow with dN/deta  = 2000, vn = 2%");
+        comment = comment.Append(" Flow with dN/delta  = 2000, vn = 2%");
```

Comment:
It's also close to `data` ... that's the problem with things that aren't recognized words, they're often midway between multiple candidates.